ERCS/erc-7730.md

```diff
@@ -0,0 +1,440 @@
+* Fields in the data to sign are tied to primitive types only, but those can be displayed in many different ways. For instance, integers can be displayed as percentages, dates, etc...
+* Some fields require additional metadata to be displayed correctly, for instance token amounts require knowledge of the magnitude and the ticker, as well as where to find the token address itself to be correctly formatted.
+
+This specification intends to provide a simple, open standard format to provide wallets with the additional information required to properly format a structured data to sign for review by users.
+
+Providing this additional formatting information requires deep knowledge of the way a smart contract or message is going to be used. It is expected that DApp developers will be the best placed to write such a file. The intent of an open standard is to only write this file once and have it work with most wallets supporting this standard.
+
+> Deployment model options:
+> - In each dapps repository (discoverability?)
+> - Foundation operated repository
+> - Ledger repository
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Simple example
+
+The following is a simple example of how to clear sign a `transfer` function call on an ERC20 contract.
+
+```json
+{
+    "$schema": "https://github.com/LedgerHQ/clear-signing-format-spec/blob/master/schemas/clear-signing-schema.json",
+
+    "context": {
+        "$id": "Tether USD",
+        "contract" : {
+            "chainId": 1,
+            "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+            "abi": "https://api.etherscan.io/api?module=contract&action=getabi&address=0xdac17f958d2ee523a2206206994597c13d831ec7"
+        }
+    }, 
+
+    "metadata": {
+        "owner": "Tether",
+        "info": {
+            "legalName": "Tether Limited",
+            "url": "https://tether.to/",
+            "lastUpdate": "2017-11-28T12:41:21Z"  
+        }
+    },
+
+    "display": {
+        "formats": {
+            "0xa9059cbb": {
+                "$id" : "transfer",
+                "intent": "Send",
+                "fields": {
+                    "_to" : {
+                        "label": "To",
+                        "format": "addressOrName"
+                    },
+                    "_value" : {
+                        "label": "Amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "$.context.contract.address"
+                        }
+                    }
+                },
+                "required": ["_to", "_value"]
+            }
+        }
+    }
+}
+```
+
+The `$schema` key refers to the latest version of this specification json schema (currently version XXX).
```

Comment:
```suggestion
The `$schema` key refers to the latest version of this specification json schema (currently version XXX <!-- TODO -->).
```

If you use HTML-style comments, the linter will ensure you replace them before going into Review.